### PR TITLE
C3 reem sprint2 side bar

### PR DESCRIPTION
**Pull request by Reem-Sharkawy**
@nadeembarakat  @mohamedhammad89 
please review , and keep in mind you have to switch with urls  to homepage until conflict is resolved

### CODE REVIEW

**Comment by abdelrahmanMaged** [comment]
reem i can not get any channels so i will be greateful if you gave me the
urls that i should access

**Comment by Reem-Sharkawy** [comment]
you type http://127.0.0.1:8000/homepage

**Comment by abdelrahmanMaged** [comment]
i get page not found please check ur urls again

**Comment by Reem-Sharkawy** [comment]
[image: Inline image 1]

**Comment by Reem-Sharkawy** [comment]
the url is working with me

File: project/tager/tager/urls.py [hunk 1]
```diff
@@ -36,7 +36,14 @@
     url(r'^register/$', 'tager_www.views.UserRegistration'),
     url(r'^homepage/$', 'tager_www.views.get_channels'),
     url(r'^viewingPosts/$', 'tager_www.views.view_checked_subchannel_posts'),
+
     url(r'^updatestatus/$', 'tager_www.views.update_status'),
+
+    url(r'^menu_posts/$', 'tager_www.views.menuForSubchannels'),
+    url(r'^channel_posts/$', 'tager_www.views.postsToChannels'),
+
```
Comment (abdelrahmanMaged) [inline project/tager/tager/urls.py:59]: @Reem-Sharkawy tb olele teege ezay that there is a url called homepage however you only added these 2 :D

**Comment by abdelrahmanMaged** [comment]
the url is not theere

**Comment by Reem-Sharkawy** [comment]
integration with main page: done 
solving conflicts: done 
 please review

**Comment by Reem-Sharkawy** [comment]
@iBebarz , please review sequence diagram from google drive 
https://drive.google.com/?tab=mo&authuser=0#folders/0B1y_ZJTDoHJ0UjB6WktzcXNkWVE

**Comment by Reem-Sharkawy** [comment]
@nadeembarakat , @iBebarz , @saraismail1234 , @mohamedhammad89 
kindly review

**Comment by Reem-Sharkawy** [comment]
#187

**Comment by Reem-Sharkawy** [comment]
#187

**Comment by Reem-Sharkawy** [comment]
check again

**Comment by Reem-Sharkawy** [comment]
@HebaElSobky

**Comment by ahmedshireef** [comment]
UMLs verified

**Pull request merged**